app/modules/sightings/schemas.py

```diff
@@ -35,3 +35,22 @@ class Meta(BaseSightingSchema.Meta):
             Sighting.created.key,
             Sighting.updated.key,
         )
+
+
+class FeaturedAssetOnlySchema(BaseSightingSchema):
+    """
+    Sighting schema for featured_asset_guid only API.
+    """
+
+    class Meta(BaseSightingSchema.Meta):
+
+        fields = BaseSightingSchema.Meta.fields + (
+            Sighting.created.key,
```

Comment:
Trivial again but BaseSightingSchema.Meta.fields already has  Sighting.created.key and Sighting.updated.key,
Same comment for the dump